### ISSUE TITLE
Wrong event-name if using curator media-action

### ISSUE THREAD

**Issue by fvanzeijl** [awcodes/filament-tiptap-editor#211]
### Filament Version

v3.0.100

### Plugin Version

V3.1

### PHP Version

PHP8.2

### Problem description

Hey,

If I enable curator as media action no image will be added to tip-tap.
I figured out curator-action fires event named "insert-media" while tip-tap listens to event named "insert-content".

I can temporary fix that by listen to "insert-media" and forward it via "insert-content" to tip-tap.
With Livewire3's On-attribute on `CreateRecord.php`/`EditRecord.php`-Component of the Resource:
```php
  #[On('insert-media')]
  public function insertMedia(string $statePath, array $media)
  {
    $this->dispatch('insert-content', type: 'media', statePath: $statePath, media: $media);
  }
```

But I think that should be fixed directly in package or am I missing something?

### Expected behavior

Forward "insert-media"-Event from curator-action to "insert-content"-Event from this package.

### Steps to reproduce

**filament-curator**
[src/Components/Modals/CuratorPanel.php](https://github.com/awcodes/filament-curator/blob/3.x/src/Components/Modals/CuratorPanel.php#L385-L390)

**filament-tiptap-editor**
[src/Actions/MediaAction.php](https://github.com/awcodes/filament-tiptap-editor/blob/3.x/src/Actions/MediaAction.php#L131-L143)

**Comment by awcodes** [comment]
You're right. I'll get this fixed soon. Thanks.

**Comment by awcodes** [comment]
This has been fixed in Curator with v3.1.2 of that plugin.

Thanks again.
